BioCro/DARPA/Gs_testing.Rmd

```diff
@@ -0,0 +1,253 @@
+  facet_wrap(~location, scales = "free_y", ncol = 1)
+```---
+title: "How does decreasing stomatal slope affect transpiration and biomass?"
+author: "Author: Kristina Riemer, Jessica Guo"
+output: github_document
+urlcolor: blue
+editor_options: 
+  chunk_output_type: console
+---
+
+## Read in libraries
+
+```{r}
+devtools::install("~/pecan/models/biocro")
+library(PEcAn.BIOCRO)
+library(ncdf4)
+library(PEcAn.all)
+library(dplyr)
+library(purrr)
+library(udunits2)
+library(ggplot2)
+library(data.table)
+```
+
+## Testing sensitivity of modeled transpiration and biomass to Gs parameters
+
+Regional runs of BioCro using the parameters from the 'high light' chamber treatment were conducted, but no differences in transpiration or biomass were found for the stomatal parameter experiment. Here, we test a range of stomatal parameter values at 3 different locations to a) isolate the sensitivity of transpiration and biomass to the stomatal slope and b) understand how this sensitivity may differ spatially due to environmental conditions. 
+
+This set of runs uses the same parameters as above for the high light treatment, except in `config.xml` the stomatal slope input value is changed from `r config$pft$photoParms$b1` to `r config_ss$pft$photoParms$b1`. 
```

Comment:
This produces an error when knitting because there's no config object at that point in the script.

Reply:
I eliminated the second section alltogether, was overlooked from the previous document.